keter-backend/src/main/java/ru/avplatonov/keter/keterbackend/initialize/Node.java

```diff
@@ -0,0 +1,95 @@
+package ru.avplatonov.keter.keterbackend.initialize;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Node {
```

Comment:
where is input and output names of node template?

Reply:
It is link to other nodes?

Reply:
And i think, m.b. send uuid from frontend? He is know him, bekause i send uuid for list of nodes.